blazegraph_load_rdf.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# ---
+# Help
+
+usage () {
+    cat <<HERE
+Usage:
+$0 {blazegraph_sparql_uri} {rdf_data_directory} > log/blazegraph_load.log 2>&1
+
+Example:
+$0 http://localhost:9999/blazegraph/namespace/kb/sparql ./data/test
+
+HERE
+}
+
+if [ "$1" == '-h' -o "$1" == '--help' ]; then
+    usage
+    exit
+fi
+
+
+# ---
+# Check the input parameters
+
+bg_sparql=$1
+
+if [ -z "${bg_sparql}" ]; then
+    echo
+    echo "ERROR: SPARQL URI is undefined."
+    echo
+    usage
+    exit 1
+fi
+
+rdf_path=$2
+
+if [ ! -d "${rdf_path}" ]; then
+    echo
+    echo "ERROR: RDF data path is not a directory: ${rdf_path}"
+    echo
+    usage
+    exit 1
+fi
+
```

Comment:
Anyhow, I'll take a look at this possibility of consolidating these scripts into one script.